src/ONE.Models.CSharp/Common/Configuration/Graphs/IGraphAxisMode.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace ONE.Models.CSharp.Models.Common.Configuration.Graphs
+{
+    public enum IGraphAxisMode
```

Comment:
the name tricked me into thinking this was an interface.  but perhaps too late to change since this was the name in the sdk